nominees/public-utility-data-liberation-project.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "Public Utility Data Liberation Project",
+  "aliases": [
+    "PUDL"
+  ],
+  "description": "The Public Utility Data Liberation (PUDL) project takes US energy system information that’s already publicly available, and makes it publicly usable, by cleaning, standardizing, and cross-linking utility data from different sources in a single database.",
+  "website": "http://catalyst.coop/pudl",
+  "license": [
+    {
+      "spdx": "CC-BY-4.0",
+      "licenseURL": "https://github.com/catalyst-cooperative/pudl"
+    },
+    {
+      "spdx": "MIT",
+      "licenseURL": "https://github.com/catalyst-cooperative/pudl/blob/main/LICENSE.txt"
+    }
+  ],
+  "SDGs": [
+    {
+      "SDGNumber": 7,
```

Comment:
Hard to disagree with this one. Almost a perfect fit.

Reply:
Resolved in commit 5298ad0